src/battle_script_commands.c

```diff
@@ -14844,12 +14844,16 @@ bool8 IsMoveAffectedByParentalBond(u16 move, u8 battlerId)
         {
             switch (GetBattlerMoveTargetType(battlerId, move))
             {
+            // Both foes are alive, spread move strikes once
             case MOVE_TARGET_BOTH:
-                if (CountAliveMonsInBattle(BATTLE_ALIVE_DEF_SIDE) >= 2) // Check for single target
+                if (IsBattlerAlive(gBattlerTarget) && IsBattlerAlive(BATTLE_PARTNER(gBattlerTarget)))
                     return FALSE;
                 break;
+            // Either both foes or one foe and its ally are alive; spread move strikes once
             case MOVE_TARGET_FOES_AND_ALLY:
-                if (CountAliveMonsInBattle(BATTLE_ALIVE_EXCEPT_ACTIVE) >= 2) // Count mons on both sides; ignore attacker
+                if (IsBattlerAlive(BATTLE_PARTNER(gBattlerTarget))
```

Comment:
I made a new condition for `CountAliveMonsInBattle` which seems to work:
```
    case BATTLE_ALIVE_EXCEPT_ATTACKER:
        for (i = 0; i < MAX_BATTLERS_COUNT; i++)
        {
            if (i != gBattlerAttacker && !(gAbsentBattlerFlags & gBitTable[i]))
                retVal++;
        }
        break;
```
![exceptAttacker](https://user-images.githubusercontent.com/2904965/197428156-67108128-8767-4b2b-9030-24198b1b252e.gif) ![exceptAttacker2](https://user-images.githubusercontent.com/2904965/197428158-46f0c372-dd61-4515-9fb4-00d9f1376f1b.gif)